src/App.jsx

```diff
@@ -2,14 +2,13 @@ import React from "react";
 import RecipesMainView from "./components/RecipesMainView";
 import RecipeFullView from "./components/RecipeFullView";
 import { Routes, Route } from "react-router-dom";
-import { Container, Typography } from "@mui/material";
+import { Container } from "@mui/material";
+import NavigationBar from "./components/NavigationBar";
 
 const App = () => {
   return (
     <Container>
-      <Typography data-cy="title" variant="h2" component="div" gutterBottom>
-        Recipe Hub
-      </Typography>
+      <NavigationBar />
```

Comment:
I would place the NavigationBar outside the container, so it will expand across all the screen width:
```
  <>
    <NavigationBar />
    <Container>
      <Routes>
        <Route path="/" element={<RecipesMainView />} />
        <Route path="recipes/:id" element={<RecipeFullView />} />
      </Routes>
    </Container>
  </>
```